admin/scripts/modules/uvod.php

```diff
@@ -98,7 +98,20 @@
 }
 
 if(post('zmenitUdaj')) {
-  dbUpdate('uzivatele_hodnoty', post('udaj'), ['id_uzivatele' => $uPracovni->id()]);
+  if(post('udaj')['login_uzivatele']) {
```

Comment:
Podmínka by tady ideálně neměla být, pokud to půjde. Jenom try/catch blok nad updatem, to je vše. Jak toho docílit viz další komentáře.